eduvarjaor/talk-text
# User interface initial structure

## Description

Created the initial user interface structure.

## **Related Issue:**

N/A

## **Main changes explained:**

- Create Vue.js initial setup with Vite.js
- Configure Tailwind CSS
- Create a recorder and attach it to the app
- Create an Openai output area

## **How to test:**

1. check into current branch
2. do `npm install` to run this PR locally
3.  run `npm run dev`

## **Screenshots or videos of changes:**

If the webpage looks like the photo, everything worked as intended.

![Screenshot_1](https://github.com/eduvarjaor/talk-text/assets/86692035/043daf8f-a21c-429a-aac7-b40430baa3a1)
